### PR TITLE
feat: add className to svg

### PR DESCRIPTION
**Pull request by maasencioh**

### CODE REVIEW

**Comment by stropitek** [comment]
Please hint to the breaking change in the semantic commit

File: src/Plot.tsx [hunk 1]
```diff
@@ -101,6 +103,8 @@ export default function Plot(props: PlotProps) {
             width={width}
             height={height}
             style={{ ...defaultSvgStyle, ...svgStyle }}
+            id={id}
+            className={className}
```
Comment (targos) [inline src/Plot.tsx:107]: should be `svgId` and `svgClassName` for consistency with the already existing `svgStyle`.

File: src/types.ts [hunk 2]
```diff
@@ -167,15 +175,18 @@ export interface MarkersProps {
...
     | ({ type: 'line' } & LineSeriesProps)
     | ({ type: 'scatter' } & ScatterSeriesProps)
   >;
   legend?: LegendProps;
-  dimensions: Dimensions;
+  dimensions?: Pick<PlotProps, 'width' | 'height' | 'margin'>;
+  svg?: Pick<
+    PlotProps,
+    'plotViewportStyle' | 'svgStyle' | 'seriesViewportStyle'
+  > & { class?: string; id?: string };
```
Comment (targos) [inline src/types.ts:189]: Here, I'd also rename `svgStyle` to `style` and use `className` because that's more "react"

Comment (maasencioh) [reply src/types.ts:189]: That was part of the discussion that we have with @stropitek, to have the object definition "framework free" (svgStyle I think that has sense to rename anyway)

Comment (targos) [reply src/types.ts:189]: even outside of a framework, the DOM property is called `className`

Comment (stropitek) [reply src/types.ts:189]: I also prefer `svg.style` over `svg.svgStyle`
I'm good with `className` didn't think about how the property was named outside of react

**Review by stropitek — APPROVED** [review]

**Review by targos — APPROVED** [review]

**Pull request merged**